### ISSUE TITLE
v1.94 (Windows): etchash@nicehash (SSL) not working, also broken with 1.95

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#374]
`TeamBlackMiner-v1.94\TBMiner.exe --algo etchash --hostname etchash.auto.nicehash.com --wallet #######################.####### --ssl-port 443 --server-passwd x --ssl --ssl-verify-none --api --api-version 1.4 --api-port 4001 --cl-devices [0]`

```
00:00:00 [2023-05-01 17:09:55.744]
00:00:00 [2023-05-01 17:09:55.807] OpenCL driver version: 527.56
00:00:00 [2023-05-01 17:09:55.828] Cuda driver version: 12.0
00:00:00 [2023-05-01 17:09:55.828] Cuda runtime API version: 12.0
00:00:00 [2023-05-01 17:09:55.828]
00:00:00 [2023-05-01 17:09:55.828]
00:00:00 [2023-05-01 17:09:55.828]
00:00:00 [2023-05-01 17:09:55.828] Welcome to Team Black Miner 1.94
00:00:00 [2023-05-01 17:09:55.828]
00:00:00 [2023-05-01 17:09:55.828]
00:00:00 [2023-05-01 17:09:55.828] GPU0 gfx1010:xnack- 8GB added to miner as OpenCL device
00:00:00 [2023-05-01 17:09:55.829] Trying stratum+ssl://etchash.auto.nicehash.com:443 (etchash)
00:00:00 [2023-05-01 17:09:55.830] Api server started at 127.0.0.1:4001
00:00:00 [2023-05-01 17:09:55.830] GPU0 Started CL work thread
00:00:00 [2023-05-01 17:09:55.877] etchash.auto.nicehash.com: Pool responded successfully to subscribe
00:00:00 [2023-05-01 17:09:55.899] etchash.auto.nicehash.com: Pool responded successfully to extranonce subscribe
00:00:00 [2023-05-01 17:09:55.915] etchash.auto.nicehash.com: Pool responded successfully to authorize
00:00:00 [2023-05-01 17:09:55.915] etchash.auto.nicehash.com: Failed mining authorize. Exiting...
00:00:00 [2023-05-01 17:09:55.915] Shutting down threads
00:00:00 [2023-05-01 17:09:55.931] Control thread exited
00:00:00 [2023-05-01 17:09:55.931] GPU0 Starting generating zil dag (Epoch: 0)
00:00:00 [2023-05-01 17:09:55.947] Stratum thread exited
00:00:00 [2023-05-01 17:09:56.475] Api thread exited

```
Non-SSL is working fine

**Comment by UselessGuru** [comment]
Still not working. Any news?

**Comment by UselessGuru** [comment]
Different error with 1.95:

> TeamBlackMiner-v1.95\TBMiner.exe --algo etchash --hostname etchash.auto.nicehash.com --wallet 33hZYX8iFg5zm9E3geqrNgeP6TMzszZk5R.Blackbox --ssl-port 443 --server-passwd x --ssl --ssl-verify-none  --api --api-version 1.4 --api-port 4006 --cuda-devices [0]
> 00:00:00 [2023-07-11 17:58:01.725]
> 00:00:00 [2023-07-11 17:58:01.788] OpenCL driver version: 531.79
> 00:00:00 [2023-07-11 17:58:01.804] Cuda driver version: 12.1
> 00:00:00 [2023-07-11 17:58:01.804] Cuda runtime API version: 12.0
> 00:00:00 [2023-07-11 17:58:01.804]
> 00:00:00 [2023-07-11 17:58:01.804]
> 00:00:00 [2023-07-11 17:58:01.804]
> 00:00:00 [2023-07-11 17:58:01.804] Welcome to Team Black Miner 1.95
> 00:00:00 [2023-07-11 17:58:01.804]
> 00:00:00 [2023-07-11 17:58:01.804]
> 00:00:00 [2023-07-11 17:58:01.898] GPU0 NVIDIA GeForce GTX 1660 SUPER 6GB added to miner as Cuda device
> 00:00:00 [2023-07-11 17:58:01.898] Api server started at 127.0.0.1:4006
> 00:00:00 [2023-07-11 17:58:01.898] Trying stratum+ssl://etchash.auto.nicehash.com:443 (etchash)
> 00:00:00 [2023-07-11 17:58:01.898] etchash.auto.nicehash.com: SSL connection to pool failed with error load_verify_file: No such file or directory (system library, fopen) [asio.ssl:33558530]
> 00:00:00 [2023-07-11 17:58:01.898] Shutting down threads
> 00:00:00 [2023-07-11 17:58:01.898] etchash.auto.nicehash.com: SSL connection to pool failed with error load_verify_file: No such file or directory (system library, fopen) [asio.ssl:33558530]

**Comment by UselessGuru** [comment]
Any news?

**Comment by sp-hash** [comment]
fixed in v1.96
